### PR TITLE
Render Changelog

### LINKED ISSUE #1217: Display CHANGELOG in the App
When an update is available, we how a dialog that has a "show changelog" button. This button currently opens https://github.com/Nexus-Mods/NexusMods.App/releases/latest in the browser. With #1216 and #1112 we can render the changelog in the App instead.

Requires #1228.

This can just be a page for now.

### PR DESCRIPTION
**Pull request by erri120**
Resolves #1217.

### CODE REVIEW

File: src/NexusMods.App.UI/Controls/MarkdownRenderer/MarkdownRendererView.axaml.cs [hunk 1]
```diff
@@ -12,12 +14,14 @@ public MarkdownRendererView()
 
         this.WhenActivated(disposables =>
         {
-            this.OneWayBind(ViewModel, vm => vm.Contents, view => view.MarkdownScrollViewer.Markdown)
-                .DisposeWith(disposables);
+            var viewModel = ViewModel!;
+
+            MarkdownScrollViewer.Engine.HyperlinkCommand = viewModel.OpenLinkCommand;
+            MarkdownScrollViewer.Engine.Plugins.PathResolver = viewModel.PathResolver;
+            MarkdownScrollViewer.Engine.Plugins.Plugins.Add(viewModel.ImageResolverPlugin);
```
Comment (Al12rs) [inline src/NexusMods.App.UI/Controls/MarkdownRenderer/MarkdownRendererView.axaml.cs:21]: Should this be in `this.WhenAnyValue(v=>v.ViewModel).WhereNotNull().Do(PopulateFromVM())` or is out of that by design?

Comment (erri120) [reply src/NexusMods.App.UI/Controls/MarkdownRenderer/MarkdownRendererView.axaml.cs:21]: I had before. Working with the markdown engine directly is already hacky enough.

Comment (erri120) [reply src/NexusMods.App.UI/Controls/MarkdownRenderer/MarkdownRendererView.axaml.cs:21]: I can re-add it, shouldn't make a difference... hopefully

Comment (Al12rs) [reply src/NexusMods.App.UI/Controls/MarkdownRenderer/MarkdownRendererView.axaml.cs:21]: You know about this better than me so up to you

**Review by Al12rs — APPROVED** [review]

File: src/NexusMods.App.UI/Controls/MarkdownRenderer/MarkdownRendererView.axaml.cs (hunk 1 above)
Comment (erri120) [reply src/NexusMods.App.UI/Controls/MarkdownRenderer/MarkdownRendererView.axaml.cs:21]: I'll keep this as is. It doesn't really make sense to react to VM changes in this context. I'd have to reset the entire markdown rendering engine and then add the new stuff.

**Pull request merged**